src/Illuminate/Encryption/composer.json

```diff
@@ -16,6 +16,7 @@
     "require": {
         "php": ">=5.5.9",
         "ext-openssl": "*",
+        "ext-mbstring": "*",
```

Comment:
I would prefer to disagree on this, we should refer this as we refer `"php": ">=5.5.9"`, why do we include it here when illuminate/support already include the same line? Clarity, someone who probably looking for an encryption package found out about this package and decide to try it out because packagist only show it require php 5.5 & openssl extension.